apps/cms.c

```diff
@@ -1012,7 +1012,7 @@ int cms_main(int argc, char **argv)
 
             pctx = CMS_RecipientInfo_get0_pkey_ctx(ri);
             if (kparam != NULL) {
-                if (!cms_set_pkey_param(pctx, kparam->param))
+                if (pctx && !cms_set_pkey_param(pctx, kparam->param))
```

Comment:
```suggestion
                if (pctx != NULL && !cms_set_pkey_param(pctx, kparam->param))
```

Reply:
This change is required to make the patch conform with the project's coding style.